go.mod

```diff
@@ -1,93 +1,105 @@
 module github.com/hashicorp/vault-plugin-secrets-gcp
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
 
 require (
 	github.com/hashicorp/errwrap v1.1.0
 	github.com/hashicorp/go-cleanhttp v0.5.2
-	github.com/hashicorp/go-gcp-common v0.8.0
+	github.com/hashicorp/go-gcp-common v0.8.1-0.20240430203852-60d57fe28861
```

Comment:
Pinned to `main`, but can also release a new tag if folks think that's more appropriate